sqlite3/lib/src/ffi/constants.dart

```diff
@@ -1,44 +1,247 @@
+
+  /// Notifications from sqlite3_log()
+  static const int SQLITE_NOTICE = 27;
+
+  /// Warnings from sqlite3_log()
+  static const int SQLITE_WARNING = 28;
 
-// Flags for file open operations, https://www.sqlite.org/c3ref/c_open_autoproxy.html
+  /// sqlite3_step() has another row ready
+  static const int SQLITE_ROW = 100;
 
-const SQLITE_OPEN_READONLY = 0x00000001;
-const SQLITE_OPEN_READWRITE = 0x00000002;
-const SQLITE_OPEN_CREATE = 0x00000004;
-const SQLITE_OPEN_URI = 0x00000040;
-const SQLITE_OPEN_MEMORY = 0x00000080;
-const SQLITE_OPEN_NOMUTEX = 0x00008000;
-const SQLITE_OPEN_FULLMUTEX = 0x00010000;
-const SQLITE_OPEN_SHAREDCACHE = 0x00020000;
-const SQLITE_OPEN_PRIVATECACHE = 0x00040000;
-const SQLITE_OPEN_NOFOLLOW = 0x01000000;
+  /// sqlite3_step() has finished executing
+  static const int SQLITE_DONE = 101;
+}
+
+/// Flags for file open operations, https://www.sqlite.org/c3ref/c_open_autoproxy.html
+/// Flags For File Open Operations
+///
+/// These bit values are intended for use in the
+/// 3rd parameter to the [sqlite3_open_v2()] interface and
+/// in the 4th parameter to the [sqlite3_vfs.xOpen] method.
+class SqlFlags {
+  /// Ok for sqlite3_open_v2()
+  static const int SQLITE_OPEN_READONLY = 0x00000001;
+
+  /// Ok for sqlite3_open_v2()
+  static const int SQLITE_OPEN_READWRITE = 0x00000002;
+
+  /// Ok for sqlite3_open_v2()
+  static const int SQLITE_OPEN_CREATE = 0x00000004;
+
+  /// VFS only
+  static const int SQLITE_OPEN_DELETEONCLOSE = 0x00000008;
+
+  /// VFS only
+  static const int SQLITE_OPEN_EXCLUSIVE = 0x00000010;
+
+  /// VFS only
+  static const int SQLITE_OPEN_AUTOPROXY = 0x00000020;
+
+  /// Ok for sqlite3_open_v2()
+  static const int SQLITE_OPEN_URI = 0x00000040;
+
+  /// Ok for sqlite3_open_v2()
+  static const int SQLITE_OPEN_MEMORY = 0x00000080;
+
+  /// VFS only
+  static const int SQLITE_OPEN_MAIN_DB = 0x00000100;
+
+  /// VFS only
+  static const int SQLITE_OPEN_TEMP_DB = 0x00000200;
+
+  /// VFS only
+  static const int SQLITE_OPEN_TRANSIENT_DB = 0x00000400;
+
+  /// VFS only
+  static const int SQLITE_OPEN_MAIN_JOURNAL = 0x00000800;
+
+  /// VFS only
+  static const int SQLITE_OPEN_TEMP_JOURNAL = 0x00001000;
+
+  /// VFS only
+  static const int SQLITE_OPEN_SUBJOURNAL = 0x00002000;
+
+  /// VFS only
+  static const int SQLITE_OPEN_MASTER_JOURNAL = 0x00004000;
+
+  /// Ok for sqlite3_open_v2()
+  static const int SQLITE_OPEN_NOMUTEX = 0x00008000;
+
+  /// Ok for sqlite3_open_v2()
+  static const int SQLITE_OPEN_FULLMUTEX = 0x00010000;
+
+  /// Ok for sqlite3_open_v2()
+  static const int SQLITE_OPEN_SHAREDCACHE = 0x00020000;
+
+  /// Ok for sqlite3_open_v2()
+  static const int SQLITE_OPEN_PRIVATECACHE = 0x00040000;
+
+  /// VFS only
+  static const int SQLITE_OPEN_WAL = 0x00080000;
+}
 
 // Prepare flags, https://www.sqlite.org/c3ref/c_prepare_normalize.html
-const SQLITE_PREPARE_PERSISTENT = 0x01;
-const SQLITE_PREPARE_NO_VTAB = 0x04;
+class SqlPrepareFlags {
```

Comment:
Same here I think?

Reply:
You are right, I've changed the visibility in sqlite3.dart on the last commit